code/modules/mob/living/carbon/human/death.dm

```diff
@@ -108,7 +108,7 @@
 	if(SSticker && SSticker.mode)
 		SSblackbox.ReportDeath(src)
 
-/mob/living/carbon/human/update_revive()
+/mob/living/carbon/human/update_revive(updating)
```

Comment:
Unsure if the argument needs to be written explicitly here to be passed to the parent, but I put it in for good measure

Reply:
Dreamchecker can throw a fit sometimes